src/Keboola/SSHTunnel/SSH.php

```diff
@@ -16,7 +16,7 @@ class SSH
 
     public function generateKeyPair()
     {
-        $process = new Process("ssh-keygen -b 2048 -t rsa -f ./ssh.key -N '' -q");
+        $process = Process::fromShellCommandline("ssh-keygen -b 2048 -t rsa -f ./ssh.key -N '' -q");
```

Comment:
Jakou to má výhodu?

Reply:
Symfony/Process ma od verzie 4 ako prvy argument pole a nie string, ale tiez ma tuto sikovnu staticku metodu, ktora berie string, prislo mi to jednoduchsie ako prepisovat vsetky tie argumenty do pola.

Reply:
https://github.com/symfony/process/blob/4.2/Process.php#L147

Reply:
Aha jasně. Takže to nění nijak chytré a je to jen workaround pro to, když to nutně potřebuješ a neescapuje, narozdíl od pole...

Reply:
aha, to je zajimavy, vypada to, ze ve 4.1 to jeste podporovalo string a ve 4.2 uz to nejde? to je dobry vedet

Reply:
jj, presne ako pises, v 4.2 uz to vyhodi "deprecated", preto som aj tu verzi v composeri zvysil na 4.2